### PR TITLE
added error message when editing cachelist from search.php

### PR DESCRIPTION
**Pull request by following5**
fix for http://redmine.opencaching.de/issues/969

I included a revert of 1931c6f here, to make the `require` statements consistent for all toplevel scripts. Ok?

### CODE REVIEW

**Comment by following5** [comment]
> I included a revert of 1931c6f here, to make the require statements consistent for all toplevel scripts.

Have removed that, becaus the require thing needs further discussion.

File: htdocs/templates2/ocstyle/search.result.caches.tpl [hunk 1]
```diff
@@ -48,6 +48,10 @@
     </div>
 {/if}
 
+{if $invalid_waypoints}
+    <p class="errormsg">{t}The following waypoints are invalid and could not be added to the list:{/t}&nbsp; {$invalid_waypoints}</p>
```
Comment (mirsch) [inline htdocs/templates2/ocstyle/search.result.caches.tpl]: XSS Attack possible. Please make sure to escape variables which come through a request.

Example Exploit: http://local.opencaching.de/search.php?searchto=searchbylist&listid=1&showresult=1&f_disabled=0&f_inactive=0&f_ignored=1&sort=byname&invalidwp=%3Cscript%20type=%22text/javascript%22%3Ealert(%27Hack%27);%3C/script%3E

Comment (following5) [inline htdocs/templates2/ocstyle/search.result.caches.tpl]: fixed

**Comment by mirsch** [comment]
👍

**Comment by teiling88** [comment]
pls squash your commits after that it is ready to merge :)

**Pull request merged**